orc8r/gateway/python/magma/ctraced/trace_manager.py

```diff
@@ -212,39 +309,29 @@ def _build_trace_filename(self):
             int(time.time()),
             _TRACE_FILE_EXT)
 
-    def _execute_start_trace_command(self, command: List[str]) -> bool:
-        """Executes a command to start a call trace
+    def _should_postprocess(self) -> bool:
+        return len(self._display_filters) > 0
```

Comment:
minor nit: len(self._display_filters) -> self._display_filters

Reply:
It's always a string